apps/catalogue/src/store/getters.js

```diff
@@ -47,6 +47,7 @@ export default {
       .conditions;
   },
   resources: (state) => state.resources,
+  cohorts: (state) => state.cohorts,
```

Comment:
why do we need this in the global store?

Reply:
I just copied how we did resources.

I don't like this 'store' business at all, but I thought to make it consistent.

Reply:
the app started as true single page app , now we have multiple view i suggest keeping the data ( maybe except for schema/lookup stuff) local in the page/view

Reply:
Well, actually it needs to be used in the detail view also. However, in my mind we have multiple apps in here because now via different routes state might become confusing.

Reply:
decided not to fix now because there are now indeed multiple components using the cohorts.
But indeed we should reconsider use of store (and if we need store use pinia or something simple like that)